src/main/java/frc/robot/subsystems/ArmSubsystem.java

```diff
@@ -16,14 +20,23 @@
 public class ArmSubsystem extends SubsystemBase {
   private CANSparkMax armMotor;
   private Solenoid brake;
-  private DutyCycleEncoder encoder;  
-  
+  private DutyCycleEncoder encoder;
+  private PIDController pid;
+
 
+ 
+  private static final int kCanID = Constants.ARM_MOTOR;
+  private static final MotorType kMotorType = MotorType.kBrushless;
+  public double  kP,kD ,kI;
```

Comment:
Lets move these PID constants to our Constants.java file as ARM_P_COEFF, ARM_I_COEFF, ARM_D_COEFF and assign them initial values which are the same as our Arm from 2020. It's always a good idea to tune your initial values based on your last similar subsystem as this gives a better starting point.

https://github.com/frc1675/frc1675-2020/blob/68af0033b8a310e56f6e53c2ebcc9e93ca42d2b9/src/main/java/frc/robot/Constants.java#L96

_Note: Only the P value was defined which means you can set I and D values to 0._